ext/enumerable/statistics/extension/statistics.c

```diff
@@ -1283,7 +1283,7 @@ enum_sum(int argc, VALUE* argv, VALUE obj)
     enum_sum_count(obj, init, skip_na, &sum, NULL);
   }
   else {
-    rb_funcall(orig_enum_sum, rb_intern("call"), argc, &init);
+    return rb_funcall(obj, id_builtin_enum_sum, argc, &init);
```

Comment:
Let's follow the existing style:

```suggestion
    sum = rb_funcall(obj, id_builtin_enum_sum, argc, &init);
```